invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -62,6 +63,8 @@ class Meta:
     created = fields.Str(dump_only=True)
     updated = fields.Str(dump_only=True)
     revision = fields.Integer(dump_only=True)
+    ui = NestedAttribute(UISchema, attribute="metadata", dump_only=True)
```

Comment:
Comment: I think this should be just the entire object not only the metadata part? You can merge and fix in another PR.